Algebraic_kernel_d/test/Algebraic_kernel_d/include/CGAL/_test_algebraic_kernel_1.h

```diff
@@ -124,9 +126,12 @@ void test_algebraic_kernel_1(const AlgebraicKernel_d_1& ak_1){
     typedef typename Name::first_argument_type AT1_;                    \
     typedef typename Name::second_argument_type AT2_;                   \
     typedef typename Name::result_type   RT_;                           \
-    {CGAL_static_assertion(( ::boost::is_same<AT1,AT1_>::value));}        \
-    {CGAL_static_assertion(( ::boost::is_same<AT2,AT2_>::value));}        \
-    {CGAL_static_assertion(( ::boost::is_same<RT,RT_>::value));}          \
+    typedef typename Name::argument_type AT1_;                          \
+    typedef typename Name::argument_type AT2_;                          \
+    typedef typename Name::result_type   RT_;                           \
```

Comment:
Does this compile? I see two different definitions for `AT1_`, `AT2_` and `RT_`?